config/src/main/java/org/hyperledger/besu/config/experimental/ExperimentalEIPs.java

```diff
@@ -55,4 +55,10 @@ public static void eip1559MustBeEnabled() {
       names = {"--Xeip1559-slack-coefficient"},
       arity = "1")
   public static Long slackCoefficient = 2L;
+
+  @Option(
+      hidden = true,
+      names = {"--Xeip1559-block-gas-limit"},
+      arity = "1")
+  public static Long blockGasLimit = 5000L;
```

Comment:
Note: this constant is not a _new_ thing, it ought to already be defined somewhere

Reply:
thank you for your comment. it was already defined in the `MainnetBlockHeaderValidator`  class.

https://github.com/hyperledger/besu/pull/2244/files#diff-043386c0ad70f6b6e23dfd43eeafce070460994e3f2140ff1be28cd93c5f2413R126